src/engine/resources.ts

```diff
@@ -397,6 +403,25 @@ export function getRunawaySources(location?: Location) {
       chance: () => 1,
       banishes: false,
     },
+    {
+      name: "Comma Chameleon",
+      prepare: (): void => {
+        if (commaItem !== undefined) {
```

Comment:
Probably also want to check you haven't currently had the comma eat something already?